### DISCUSSION TITLE
Improve `PresentSkins` by adding ability to view and select imported skins

### DISCUSSION THREAD

**Discussion by Cootz** [ppy/osu#22638] [Ideas] [answered]
View and select from imported skins by clicking on import notification.
Showcase:

https://user-images.githubusercontent.com/50776304/218560895-26d1c18f-f2e1-419d-aae3-369260013957.mp4

Ideas:
- Instantly apply skin if only one was imported (Already merged to master)
- Navigate to skin section in setting by click (by @Bruno5430 )
- Highlight added skins in the dropdown, or add a marker to show that they are new, or move them to the top above the default alphabetical ordering (by @peppy )
- Use osu-web-like notifications (by @agatemosu )

Original version of PR #22629 cannot provide stable version of this due to lack of testing and *unsafe* code in general. Looking forward would be really nice to add this later. If you have any ideas on this bring it up!

**Comment by peppy** [comment]
See https://github.com/ppy/osu/discussions/22631#discussioncomment-4969118 as it may also provide a good solution we can apply to all imports.

**Comment by bdach** [comment, marked as answer]
Mostly addressed by https://github.com/ppy/osu/pull/22629
